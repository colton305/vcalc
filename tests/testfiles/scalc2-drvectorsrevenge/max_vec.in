print(((0-2)..214748364)[214748367]);
print(((0-2)..214748364)[214748366]);

//CHECK_FILE:max_vec.out
