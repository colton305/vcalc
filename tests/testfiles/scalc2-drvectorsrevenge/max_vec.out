0
214748364
